File: Tutorial1-1Basics/src/main/java/com/example/roompersistencelibrary/data/AppDatabase.java
package com.example.roompersistencelibrary.data;

import android.arch.persistence.room.Database;
import android.arch.persistence.room.RoomDatabase;

/**
 * Entities are tables in database
 */
@Database(entities = {User.class}, version = 1)
public abstract class AppDatabase extends RoomDatabase {
    public abstract UserDao userDao();
}

File: Tutorial1-1Basics/src/main/java/com/example/roompersistencelibrary/data/AppDatabase.kt
package com.example.roompersistencelibrary.data

import androidx.room.Database
import androidx.room.RoomDatabase

/**
 * Entities are tables in database
 */
@Database(
    entities = [User::class],
    version = 1
)
abstract class AppDatabase : RoomDatabase() {
    abstract fun userDao(): UserDao?
}